udata/templates/territories/search-result.html

```diff
@@ -1,14 +1,21 @@
 {% cache cache_duration, 'territory-search-result', territory.id, g.lang_code %}
 {% set logo=territory.logo_url() or theme_static('img/placeholder_territory.png') %}
+{% if territory.level_name == 'town' %}
```

Comment:
I want a better solution here but I'm not sure where to put the `gettext` call in order to be found during the parsing given that it's dynamic.